src/wallet/rpcdump.cpp

```diff
@@ -151,8 +154,23 @@ UniValue importprivkey(const JSONRPCRequest& request)
         // whenever a key is imported, we need to scan the whole chain
         pwalletMain->UpdateTimeFirstKey(1);
 
+        /*
+         * use this instead of the genesis block
```

Comment:
Not sure about the comment, it feels as if it describes the change, not the code.

Reply:
Yeah, that might be a vestigial comment. I don't see much value in keeping it.